tests/test_instruments.py

```diff
@@ -63,11 +63,12 @@ def sync():
 
 
 def test_gpu_backend():
-    assert list(sorted(get_backends())) == ["cuda", "rocm"]
+    assert list(sorted(get_backends())) == ["cpu", "cuda", "rocm"]
 
 
 def test_gpu_info_cpu():
-    assert get_gpu_info(arch="cpu") == {"arch": "cpu", "gpus": {}}
+    smi = select_backend()
+    assert get_gpu_info(smi) == {"arch": "cpu", "gpus": {}}
```

Comment:
Could we keep the original interface for `get_gpu_info`? It is a bit simpler and it is used in multiple places in milabench.

Reply:
done